cmd/clusteringress/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusteringress
```

Comment:
This kind of limits us to a single resource for this controller.  What if we introduce a namespaced Ingress concept?

Perhaps we should call this `networking` and it should have an OWNERS file to match?

Reply:
+1 I should've caught this :)

@greghaynes please use this same OWNERS file https://github.com/knative/serving/blob/master/pkg/reconciler/v1alpha1/clusteringress/OWNERS

Reply:
I think I got this right - I also renamed the deployment and properties to be networking-istio rather than clucteringress-istio